source/irfs/full_enclosure/psf/psf_table/index.rst

```diff
@@ -18,9 +18,9 @@ Columns:
 * ``ENERG_LO``, ``ENERG_HI`` -- ndim: 1, unit: TeV
     * True energy axis
 * ``THETA_LO``, ``THETA_HI`` -- ndim: 1, unit: deg
-    * Field of view offset axis (see :ref:`coords-fov`)
+    * Angular distance to centre of the array pointing direction (see :ref:`coords-fov`)
```

Comment:
AE vs BE? Do we have a guideline for that? I would say AE

Reply:
Please keep "axis", as this refers to the axis definition of the matrix

Reply:
I would not say "Array", as this may be used by non-IACT instruments...

Reply:
But then some one should change https://gamma-astro-data-formats.readthedocs.io/en/latest/general/coordinates.html#coords-fov , as here it is explicitly said that these types of coordinates are used for IACTs and that the centre of the FOV is the centre of the array pointing direction.

Reply:
@TarekHC I would also say that the issue of rewording everything so it also works e.g. for HAWC is a larger one and should probably be done by someone from the Water Cherenkov community.

Reply:
There are too many places where this is currently an issue.